### PR TITLE
Add signing 2.0 rebase

### PR DESCRIPTION
**Pull request by MattHardcastle**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21193/HighFidelity-Beta-PR16150-768ac08.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21193/HighFidelity-Beta-Interface-PR16150-768ac08.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21193/HighFidelity-Beta-PR16150-768ac08.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21193/HighFidelity-Quest-Beta-PR16150-768ac08.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21193/HighFidelity-Beta-PR16150-768ac08.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21193/HighFidelity-Beta-Interface-PR16150-768ac08.exe

File: cmake/macros/SetPackagingParameters.cmake [hunk 1]
```diff
@@ -38,7 +38,7 @@ macro(SET_PACKAGING_PARAMETERS)
     set(BUILD_ORGANIZATION "High Fidelity")
     set(HIGH_FIDELITY_PROTOCOL "hifi")
     set(HIGH_FIDELITY_APP_PROTOCOL "hifiapp")
-    set(INTERFACE_BUNDLE_NAME "Interface")
+    set(INTERFACE_BUNDLE_NAME "interface")
```
[the remark this answers is not in the document]
Comment (danteruiz) [reply cmake/macros/SetPackagingParameters.cmake]: like what?

[the remark this answers is not in the document]
Comment (MattHardcastle) [reply cmake/macros/SetPackagingParameters.cmake]: Launcher always uses lowercase when referencing "interface.app"

**Pull request merged**